### ISSUE TITLE
Standalone not working for me.

### ISSUE THREAD

**Issue by AlanEdd** [Mfarooq360/WinThumbsPreloader-V2#12]
Hey there,
The standalone (for me anyway) seems to just open the welcome screen without doing the job.
I have tried all combinations of flags. I also tried removing the final backslash in the paths.

"pathTo...\WinThumbsPreloader.exe" -r -s "D:\OneDrive - RTÉ\Preview\Old\"

Just popping this up:
![image](https://github.com/Mfarooq360/WinThumbsPreloader-V2/assets/36997610/e103cab1-554d-4ae0-b032-5c0b2fbce407)

**Comment by VL4DST3R** [comment]
It may be an issue with special characters (in your case that `É`). Can you try on a path without it?

**Comment by AlanEdd** [comment]
Yes! that seems to work, thank you so much. 

[path to winthumbspreloader.exe can't have non standard characters]

As a general question about this software: It preloads the thumbnails, does that mean that it loads them into RAM for the session and that it is flushed on restarting the machine ?
I see how it works but don't fully understand what is happening behind the scenes... 

Thank you ! 
Alan.

**Comment by Mfarooq360** [comment]
> 2. Press `Preload`

I'd also like to mention that it might not be working here since it doesn't search subfolders by default, but if you hold shift before clicking, it should change to `Recursively`, which should allow it to search all subfolders of the selected path.

**Comment by Korb** [comment]
> if you hold **shift** before clicking, it should change to `Recursively`

Yes! Thank you! https://github.com/Mfarooq360/WinThumbsPreloader-V2/discussions/3#discussioncomment-15048033

**Comment by Korb** [comment]
> I'm working on a fix for the next version.

@VL4DST3R, considering that almost a year has passed since the last beta was released, should we understand that the development of the app is frozen?

**Comment by VL4DST3R** [comment]
Hey! Almost certain it's not frozen, it's just that Mutahar's been busy with life, like everyone else, and is working on it only when he gets the time to do so. If you want a more recent status update check [this issue](https://github.com/Mfarooq360/WinThumbsPreloader-V2/issues/25#issuecomment-4910032036)

**Comment by Mfarooq360** [comment]
[Beta 7](https://github.com/Mfarooq360/WinThumbsPreloader-V2/releases/tag/v2.0.0-beta7) has now been released and should fix most of the problems with paths in the argument parser, along with popping up an error message when an invalid path is passed.
